docs/cli.md

```diff
@@ -804,4 +815,18 @@ gcli column update --name catalog_postgres.hr.departments.value --rename values
 gcli column update --name catalog_postgres.hr.departments.values --datatype "varchar(500)"
 gcli column update --name catalog_postgres.hr.departments.values --position name
 gcli column update --name catalog_postgres.hr.departments.name --null=true
```

Comment:
although this is not he change in this PR, but it might be a bug: "--null=true", should it be "--null true"?

Reply:
Both will work but I should make it consistant